source/upgrade/v3.txt

```diff
@@ -97,7 +97,16 @@ Version 3.0 Breaking Changes
   To configure logging in your application, see the :ref:`csharp-logging` guide.
 
 - The LINQ2 provider has been removed from this version of the driver.
-  You must use LINQ3 for all LINQ queries.
+  You must use LINQ3 for all LINQ queries. This change has the following effect on the
+  driver:
+
+  - Queries that use client-side projections will throw an ``ExpressionNotSupportedException``
+    by default with v3.0. To enable client-side projections, set the
```

Comment:
S: add "error"; also since this is in the v3.0 section, I don't think you need to specify that again
```suggestion
  - Queries that use client-side projections will throw an ``ExpressionNotSupportedException`` error
    by default. To enable client-side projections, set the
```